source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,53 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename InputKeyIt, typename InputValueIt,
```

Comment:
Since, unlike all other described algorithms, there is no output iterators among parameters, perhaps it makes sense to drop "Input" from the parameter names - i.e., to rename as `KeyIt` and `ValueIt`.

Reply:
Thanks, done.